.chloggen/enable_tls.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Enable mTLS for internal components
```

```suggestion
note: Enable mTLS by default for internal components
```

I'd add "by default" so it's clear this change is just a change to the default settings, and that it can also be disabled.